jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -1162,6 +931,101 @@ ecma_op_function_call_external (ecma_object_t *func_obj_p, /**< Function object
   return ret_value;
 } /* ecma_op_function_call_external */
 
+/**
+ * Declaration of ecma_op_bound_function_get_argument_list
+ *
+ * @return target function of the bound function
+ */
+static ecma_object_t *
+ecma_op_bound_function_get_argument_list (ecma_object_t *func_obj_p, ecma_collection_t *list_p);
+
+/**
+ * Append the bound arguments into the given collection
+ *
+ * Note:
+ *       - The whole bound chain is resolved
+ *       - The first element of the collection contains the bounded this value
+ *
+ * @return target function of the bound function
+ */
+static ecma_object_t * JERRY_ATTR_NOINLINE
```

Comment:
JERRY_ATTR_NOINLINE should be placed at the beginning, can't be after * character.
See #3347 for details, we ran into this strange windows build fail previously.

Reply:
Thank you for the investigation! :)